plugins/src/android/optimove.gradle

```diff
@@ -14,12 +14,12 @@ android {
 }
 dependencies{
     implementation 'com.optimove.android:optimove-android:6.+'
-       def work_version = "2.6.0"
-    // Force WorkManager 2.6.0 for transitive dependency
-    implementation("androidx.work:work-runtime-ktx:$work_version") {
-        force = true
+   
+    api(group: "androidx.work", name: "work-runtime") {
+    version {
+        strictly "2.7.0-alpha04"
```

Comment:
?

Reply:
soo, the story is 

1. latest cordova cli (11) pins cordova-android platform to 10.1.2 https://cordova.apache.org/news/2021/12/21/cordova-cli-11.0.0-release.html
2. there is cordova-android 11 released on 12 of july 2022. You can always update platform by doing `cordova platform update android@11.0.0`, but it's not the default
3. cordova-android 11.* has following gradle defaults. https://github.com/apache/cordova-android/blob/master/framework/cdv-gradle-config-defaults.json . Cordova-android 10.* has SDK_VERSION set to 30. This translates to compileSdkVersion:30
4. work-runtime > "2.7.0-alpha04" has compileSdkVersion:31.
5. This produces following exception

```
 The minCompileSdk (31) specified in a
     dependency's AAR metadata (META-INF/com/android/build/gradle/aar-metadata.properties)
     is greater than this module's compileSdkVersion (android-30).
     Dependency: androidx.work:work-runtime:2.7.0
```

so, eventually this version pinning can be removed, but think not yet? Or new projects would have the error until android platform is upgraded

Reply:
to not depend on an alpha dependency new plan is to update documentation asking to add 

```
<preference name="android-targetSdkVersion" value="31" />
```

to config.xml